charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
             {{- toYaml .Values.lifecycle | nindent 12 }}
           ports:
             - name: http
-              containerPort: 9002
+              containerPort: {{ .Values.service.containerPort }}
```

Comment:
is it possible to specify a default inline here to 9002? I'm actually unsure :)

Reply:
![Screenshot 2023-11-07 at 9 11 38 AM](https://github.com/acryldata/datahub-helm/assets/84748107/33442c12-73dc-4023-b27f-228483305316)